patch-package
--- a/node_modules/adaptivecards/lib/card-elements.js
+++ b/node_modules/adaptivecards/lib/card-elements.js
@@ -1,4 +1,7 @@
 "use strict";
+
+var twemoji = require('../../../built/lib').twemoji
+
 var __extends = (this && this.__extends) || (function () {
     var extendStatics = Object.setPrototypeOf ||
         ({ __proto__: [] } instanceof Array && function (d, b) { d.__proto__ = b; }) ||
@@ -1580,7 +1583,7 @@ var ActionButton = /** @class */ (function () {
         },
         set: function (value) {
             this._text = value;
-            this._element.innerText = this._text;
+            this._element.innerHTML = twemoji.parse(this._text);
             this._element.setAttribute("aria-label", this._text);
         },
         enumerable: true,
--- a/node_modules/adaptivecards/lib/text-formatters.js
+++ b/node_modules/adaptivecards/lib/text-formatters.js
@@ -1,4 +1,7 @@
 "use strict";
+
+var twemoji = require('../../../built/lib').twemoji
+
 var __extends = (this && this.__extends) || (function () {
     var extendStatics = Object.setPrototypeOf ||
         ({ __proto__: [] } instanceof Array && function (d, b) { d.__proto__ = b; }) ||
@@ -62,6 +65,9 @@ function formatText(lang, text) {
     for (var i = 0; i < formatters.length; i++) {
         result = formatters[i].format(lang, result);
     }
+
+    result = "<span>"+twemoji.parse(result)+"</span>"
+
     return result;
 }
 exports.formatText = formatText;
